CHANGELOG/preview.md

```diff
@@ -1,5 +1,126 @@
 # Preview Changelog
 
+## [7.6.0-preview.4] - 2025-03-27
+
+### Breaking Changes
+
+- Fix `WildcardPattern.Escape` to escape lone backticks correctly (#25211) (Thanks @ArmaanMcleod!)
+- Convert `-ChildPath` parameter to `string[]` for `Join-Path` cmdlet (#24677) (Thanks @ArmaanMcleod!)
+
+### Engine Updates and Fixes
+
+- Add `PipelineStopToken` to `Cmdlet` which will be signaled when the pipeline is stopping (#24620) (Thanks @jborean93!)
+- Fallback to AppLocker after `WldpCanExecuteFile` (#24912)
+- Move .NET method invocation logging to after the needed type conversion is done for method arguments (#25022)
+- Fix share completion with provider and spaces (#19440) (Thanks @MartinGC94!)
+
+### General Cmdlet Updates and Fixes
+
+- Exclude outvariable assignments within the same CommandAst when inferring variables (#25224) (Thanks @MartinGC94!)
```

Comment:
```suggestion
- Exclude `-OutVariable` assignments within the same `CommandAst` when inferring variables (#25224) (Thanks @MartinGC94!)
```

Reply:
Should we use the format rule for PR naming or it is not good for GitHub search?

Reply:
@iSazonov unless something changed with search the inclusion of hyphens/dashes makes it hard for getting good search results for things like cmdlet/function names & parameters like this